Example/PaymentSheet Example/PaymentSheet Example/PaymentSheetTestPlayground.swift

```diff
@@ -209,14 +213,11 @@ class PaymentSheetTestPlayground: UIViewController {
     }
 
     var initMode: InitMode {
```

Comment:
Hm "initialization mode" is not the right name for these things now.  It's more like "integration type" or something.

Or maybe:
Initialization mode: [Normal, Deferred]
Deferred server mode: [Create, Create & Confirm, Multiprocessor]

Not sure, wdyt?